js/classes/zone.js

```diff
@@ -135,33 +135,22 @@ class Zone {
          */
 
         return {
-            factory: {
-
-            },
-            lava: {
-
-            },
-            mountain: {
-
-            },
```

Comment:
I don't think it's a good idea to put all of the zones under village and delete these categories. Having categories like `village`, `mountain`, `forest`, etc will keep it much more organized.

Reply:
We can add those back, I was thinking that it would make sense to get rid of them because we won't have that many zones.